<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaultInjectionClass</class>
 <widget class="QWidget" name="FaultInjectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaultInjection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>注入故障的设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cBDeviceType">
       <item>
        <property name="text">
         <string>BMU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BCU</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>从机ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>     注入类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>电压注入故障</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>温度注入故障</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AFE注入故障</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>风扇转速注入故障</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DI注入故障</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>电流注入故障</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_clear">
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_read_all">
       <property name="text">
        <string>读取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_write">
       <property name="text">
        <string>下设全部</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
